test/unit/test_exports/test_derived_quantities/test_surface_flux.py

```diff
@@ -95,7 +96,8 @@ def test_h_flux_with_soret(self):
 @pytest.mark.parametrize("height", [2, 3])
 @pytest.mark.parametrize("c_top", [2, 3])
 @pytest.mark.parametrize("c_bottom", [2, 3])
-def test_compute_cylindrical(r0, radius, height, c_top, c_bottom):
+@pytest.mark.parametrize("Soret", [False, True])
+def test_compute_cylindrical(r0, radius, height, c_top, c_bottom, Soret):
```

Comment:
Let's follow the [PEP8 recommendations](https://peps.python.org/pep-0008/#function-and-variable-names:~:text=Function%20names%20should,retain%20backwards%20compatibility.) here and use lowercase

```suggestion
@pytest.mark.parametrize("soret", [False, True])
def test_compute_cylindrical(r0, radius, height, c_top, c_bottom, soret):
```